apps/charterafrica/src/theme/index.js

```diff
@@ -195,6 +195,45 @@ deepmerge(
         },
       },
     },
+    MuiInput: {
+      styleOverrides: {
+        root: {
+          ...theme.typography.h6Small,
+          border: "1px solid",
+          borderRadius: "4px",
+          borderColor: neutral[400],
+        },
+        sizeSmall: {
+          height: "36px",
+        },
+        sizeMedium: {
```

Comment:
@kilemensi Not sure if this should be here as they do not appear on the [input api](https://mui.com/material-ui/api/input/#css), only `sizeSmall` is present. If not what would be the approach to style medium and large inputs in the theme?

Reply:
_Heita_ @kelvinkipruto 

1. I think we should focus on [`MuiOutlinedInput`](https://mui.com/material-ui/api/outlined-input/) first since that's the `TextField` variant that closely resembles the designed `Input` component. Only switch `MuiInputBase` for props/styles directly inherited from it.
2. From the API [docs](https://mui.com/material-ui/api/text-field/#props), `TextField` supports `medium` and `small` sizes. This means `large` is the **_only_** one that doesn't have direct support. Nevertheless, can you try setting size to `large` (since they say the `size` prop supports any `string`) and see what class name does MUI generate? 🤞🏽 it will be `sizeLarge`.